frontend/packages/kubevirt-plugin/integration-tests-cypress/utils/const/index.ts

```diff
@@ -85,6 +86,15 @@ export enum DISK_DRIVE {
   CDROM = 'CD-ROM',
 }
 
+export enum K8SKIND {
```

Comment:
nitpick: `K8SKIND` => `K8S_KINDS`

Reply:
NP

Reply:
That works! Both singular and plural names are used for enums throughout the codebase, but I think you're right that it should be singular.